apis/apis.json

```diff
@@ -470,9 +470,11 @@
 
   {
     "id": "Google.Cloud.Redis.V1Beta1",
+    "productName": "Google Cloud Memorystore Redis",
```

Comment:
OK, probably here really

Reply:
This was the name given to me by the team, so I'll stick with it for now.

Reply:
(Except that that's not what the web site says, so I'll match that.)